images/python-openstackclient/Dockerfile

```diff
@@ -17,7 +17,8 @@ uv pip install \
         python-neutronclient \
         python-octaviaclient \
         python-openstackclient \
-        python-swiftclient
+        python-swiftclient \
+        tap-as-a-service
```

Comment:
Is the client actually located in the main `tap-as-a-service` package?  So we have to pull in the entire service to get access to the CLI?

Reply:
gross, but ok